contributed_definitions/nyaml/NXelectron_level.yaml

```diff
@@ -0,0 +1,1465 @@
+category: base
+doc: |
+  Electronic level probed in X-ray spectroscopy or resonance experiments.
+type: group
+NXelectron_level(NXobject):
+  element:
+    doc: |
+      Symbol of the chemical element.
+      
+      For each, the atomic number, common English name, and standard atomic weight are also given.
```

Comment:
I wanted to align with the developments on [NXxas](https://github.com/nexusformat/definitions/pull/1293) in the main NIAC repo, so I took the same atomic_weights.

I went with `element` here because the given names are the ones given for the elements in the periodic table.

Reply:
ok